websqrl/components/WikipediaEventStory.tsx

```diff
@@ -0,0 +1,16 @@
+import { Block } from "jsxstyle";
+import type { Result } from "../src/types";
+
+export const WIKIPEDIA_FETCH_FEATURES = [
+  "AuthorUsername",
+  "TweetText",
+  "TweetId",
+  "TweetDate",
+] as const;
+export type WikipediaFetchFeature = typeof WIKIPEDIA_FETCH_FEATURES[number];
+
+type WikipediaEventStoryProps = Omit<Result<WikipediaFetchFeature>, "type">;
+
+export const WikipediaEventStory: React.FC<WikipediaEventStoryProps> = () => {
+  return <Block>hello!</Block>;
+};
```

Comment:
this is a placeholder at the moment. The wikipedia demo is broken as of a few commits ago. I briefly tried to fix it but there's some web worker funkiness going on.

Reply:
Yeah, there's a lot of pieces missing of that demo. I'm treating it as dead for now, but not too hard to revive.